{
    "type": "bundle",
    "id": "bundle--73b8aa60-7fd7-4221-8c6f-b77339e1cb06",
    "spec_version": "2.0",
    "objects": [
        {
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "source_ref": "malware--a19c1197-9414-46e3-986f-0f609ff4a46b",
            "target_ref": "attack-pattern--d63a3fb8-9452-4e9d-a60a-54be68d5998c",
            "external_references": [
                {
                    "source_name": "CERT-FR PYSA April 2020",
                    "url": "https://www.cert.ssi.gouv.fr/uploads/CERTFR-2020-CTI-003.pdf",
                    "description": "CERT-FR. (2020, April 1). ATTACKS INVOLVING THE MESPINOZA/PYSA RANSOMWARE. Retrieved March 1, 2021."
                }
            ],
            "description": "[Pysa](https://attack.mitre.org/software/S0583) has deleted batch files after execution. (Citation: CERT-FR PYSA April 2020) ",
            "relationship_type": "uses",
            "id": "relationship--fc0b1fb2-f987-4d9c-8470-c40c517925d9",
            "type": "relationship",
            "modified": "2021-04-27T03:33:35.204Z",
            "created": "2021-04-27T03:33:35.204Z"
        }
    ]
}